### ISSUE TITLE
[risk-score-agent] Skip screening score process when patient is already scored

### ISSUE THREAD

**Issue by nsuthison** [gu-tum-gun-aeng/med4all-risk-score-agent#2]
Skip risk score process when the isBypassScreening flag is true.

**Comment by catflyflyfly** [comment]
This issue is blocked by https://github.com/gu-tum-gun-aeng/med4all-be/issues/38 and https://github.com/gu-tum-gun-aeng/med4all-patient-api/issues/9.

**Comment by dodadoa** [comment]
@catflyflyfly why open this one again ?

**Comment by catflyflyfly** [comment]
I thought I just reference this one from another issue? This issue should still be in "Reviewer Approve"

**Comment by dodadoa** [comment]
I c
